pkg/reconciler/v1alpha1/taskrun/taskrun.go

```diff
@@ -191,7 +203,7 @@ func (c *Reconciler) reconcile(ctx context.Context, tr *v1alpha1.TaskRun) error
 		}
 
 		// Build is not present, create build
-		build, err = c.createBuild(tr, pvc.Name)
+		build, err = c.createBuild(ctx, tr, pvc.Name)
```

Comment:
what if, instead of wrapping the `image` configuration in a context, we have an attribute on the TaskRun controller to hold the path to the image?

e.g.

```go
type Reconciler struct {
	*reconciler.Base		*reconciler.Base
 	...
         entrypointImage                 string
```